### PR TITLE
[flake8] Fix flake8 errors

### PR DESCRIPTION
**Pull request by vchrombie**
This PR fixes the existing flake8 errors.

### CODE REVIEW

File: kidash/kidash.py [hunk 1]
```diff
@@ -1118,8 +1127,8 @@ def get_dashboard_name(panel_file):
     if kibana and 'dashboard' in kibana:
         dash_name = kibana['dashboard']['id']
     elif kibana:
-        logger.error("Wrong panel format (can't find 'dashboard' or 'index_patterns' fields): %s",
-                     panel_file)
+        logger.error("Wrong panel format (can't find 'dashboard' or "
+                     "'index_patterns' fields): %s", panel_file)
```
Comment (vchrombie) [inline kidash/kidash.py]: instead of such a format, we can adapt a different multiline too like
```python
        logger.error("""Wrong panel format (can't find 'dashboard' or 
                     index_patterns' fields): %s""", panel_file)
```

WDYT?

File: kidash/kidash.py [hunk 2]
```diff
@@ -1588,45 +1636,52 @@ def fetch_dashboard(elastic_url, dash_id, es_index=None):
...
     with open(export_file, 'w') as f:
         if not split_index_patterns:
             f.write(json.dumps(dash_json, indent=4, sort_keys=True))
         else:
             index_patterns = dash_json.pop("index_patterns")
 
-            with open(export_file, 'w') as f:
-                f.write(json.dumps(dash_json, indent=4, sort_keys=True))
+            with open(export_file, 'w') as fa:
+                fa.write(json.dumps(dash_json, indent=4, sort_keys=True))
```
Comment (vchrombie) [inline kidash/kidash.py:1599]: since `f` is already in use, changing this file object to `fa`

File: kidash/kidash.py [hunk 3]
```diff
@@ -1588,45 +1636,52 @@ def fetch_dashboard(elastic_url, dash_id, es_index=None):
...
+                    raise RuntimeError("%s exists. Remove it before running."
+                                       % export_file_index)
 
                 index_pattern['value'][RELEASE_DATE] = dt.utcnow().isoformat()
                 index_pattern_importable = {"index_patterns": [index_pattern]}
-                with open(export_file_index, 'w') as f:
-                    f.write(json.dumps(index_pattern_importable, indent=4, sort_keys=True))
+                with open(export_file_index, 'w') as fb:
+                    fb.write(json.dumps(index_pattern_importable, indent=4,
+                                        sort_keys=True))
```
Comment (vchrombie) [inline kidash/kidash.py]: same as above

**Review by vchrombie — COMMENTED** [review]
Hi @sduenas 
I had a few suggestions/doubts with this PR, so opening some comments about them.

Feel free to suggest any changes.

File: kidash/kidash.py [hunk 4]
```diff
@@ -789,8 +800,7 @@ def get_index_pattern_json(elastic, index_pattern_id):
 def get_search_from_vis(elastic, vis):
     search_id = None
     vis_json = get_vis_json(elastic, vis)
-    if not vis_json:
-        search_id
```
Comment (vchrombie) [inline kidash/kidash.py:793]: Also, removed this completely as this seems to have no effect.

File: kidash/kidash.py (hunk 1 above)
Comment (sduenas) [reply kidash/kidash.py]: Multiline format will add a new line and tabs at the beginning of the second line, so it will be hard to parse in case we need it.

File: kidash/kidash.py (hunk 2 above)
Comment (sduenas) [reply kidash/kidash.py:1599]: Good!

File: kidash/kidash.py (hunk 3 above)
Comment (sduenas) [reply kidash/kidash.py]: Good!

File: kidash/kidash.py (hunk 4 above)
Comment (sduenas) [reply kidash/kidash.py:793]: Makes sense.

**Review by sduenas — COMMENTED** [review]
LGTM. Just a minor comment before we merge it. Do you think you can add the same `MANIFEST.in` we have in other repositories such in Perceval? It has some `flake8` configuration that we should include here too.

**Review by sduenas — APPROVED** [review]
LGTM
